src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/Filter/search_specification.rst

```diff
@@ -60,9 +60,7 @@ Naming
 ~~~~~~
  - Elasticsearch fields for attribute follow this naming scheme:
 
-``attribute_code-l-locale-s-scope-es_suffix``
-
-Please note that locale code is prefixed by ``l-`` and scope code is prefixed by ``s-``, to avoid collision between them.
+``attribute_code-locale-scope-es_suffix``
```

Comment:
attribute_code-es_suffix.channel.locale

if not scopable => "<all_channels>"
if not localizable => "<all_locales>"